owltorrent-network/src/main/java/edu/rice/owltorrent/network/PeerConnector.java

```diff
@@ -21,7 +21,7 @@
   protected final Peer peer;
   // TODO: bad practice, should eventually refactor
   protected final TorrentManager manager;
-  protected final StorageAdapter storageAdapter;
+  protected StorageAdapter storageAdapter;
```

Comment:
ClientHandler gets new peerConnector and store it to TorrentManage.

ClientHandler does not have access to storageAdapter do this field needs to be set later.